dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -312,10 +312,17 @@ def run_validation(self, job):
                         valid = True
                     else:
                         if file_type == "fabs":
+                            # Create afa_generated_unique
                             record['afa_generated_unique'] = (record['award_modification_amendme'] or '-none-') + "_" +\
                                                              (record['awarding_sub_tier_agency_c'] or '-none-') + \
                                                              "_" + (record['fain'] or '-none-') + "_" + \
                                                              (record['uri'] or '-none-')
+                            # Create unique_award_key
+                            identifier = record['fain'] if str(record['record_type']) == '1' else record['uri']
+                            record['unique_award_key'] = (record['record_type'] or '-none-') + '_' + \
```

Comment:
The `record_type` here would be 1-3, in the script it's `NON` or `AGG`, we should pick one.